src/Library/demos/GTK Switch/main.blp

```diff
@@ -0,0 +1,55 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "GTK Switch";
+
+  Box {
+    orientation: vertical;
+
+     Label {
+        label: _("GtkSwitch is a “light switch” that has two states: on or off.\nThe user can control which state should be active by clicking the empty area, or by dragging the handle.\n");
+        justify: center;
+        margin-bottom: 15;
+        styles ["body"]
+      }
```

Comment:
I think we don't need this
Given how visual/interactive Workbech is – we don't need to spell things out as much as regular documentation